core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.html

```diff
@@ -1,21 +1,24 @@
 @if (values.length) {
-    <div class="dot-category-field__categories" data-testId="category-chip-list">
-        @for (category of values; track category.id) {
-            <p-chip
-                [pTooltip]="category.value"
-                [removable]="true"
-                [label]="category.value"
-                tooltipPosition="top"
-                styleClass="p-chip-sm" />
-        }
-    </div>
+<div class="dot-category-field__categories" data-testId="category-chip-list">
+    @for (category of values; track category.id) {
+    <p-chip
+        [pTooltip]="category.value"
+        [removable]="true"
+        [label]="category.value"
+        tooltipPosition="top"
+        styleClass="p-chip-sm" />
+    }
+</div>
```

Comment:
@oidacra I think there are still problems because according to the official format there should be a space inside if or for blocks with tabs or spaces but with this space but in your code this space is not there.



![Screenshot 2024-06-12 at 12 08 59 PM](https://github.com/dotCMS/core/assets/7611944/3595e921-0c9b-491a-8a2a-c8701eaea171)


Using the Prettier [playground](https://deploy-preview-15606--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEABAlgMwAQAoBuAhgDYCucAzgHTEIDmMAFgJTbAA6UAPACbr7YwxQhQoBediB4QYAWjCF4dCACcAnrMzo4xHgH09Cpau0VJ2HosKz4FGAEkeE8IrjL18xugAOs4ujtJAD5ObDDsVExVPCM3VTVsCBwiMkoAbmwYFUIwAGtBV3c1KnQeVg4ocOwuXzAvb1Cq8IBtbwAVCAhiGB8AXWdYoqoU8klGpuxmlTgAWwgiACNafsks0ZBxpubhBZ0Vl2N1YZJ1zaqYTu6fAAUICnQe6GcLho3Kiew7NVoAYWFRZy1eqyCgzcwAehC7zCAF9OFxwXx8FCYSAADQgCDeR5QCjIUCEFQqCAAd2uhIQeJQJBJhDUeIxC2yeTgMAAyoQZnAADLoKBwZCYEgUOCM5m5Vls7w5Pl0ZBrUUgEUzdDylTkDH3KB0WgARVIMgFSCFxBFGIAVhQAB5s2V6g3wQXCxUARwdcGuxO8VJAIlk-LgPED6JAWUI6H82p+EBmM0IyF9xGIIa1OrgAEEYFl0AtSPBrnAVLz+U7TYrGDAZsQAOpeWzSsBwNmUh78B5qBNgUQh-DkexQIOwNlgFQ+GDpgdsmDfI0ms0gbzEkXV7LeBOLyiF-ACjF8kUqGCewh0OOl+fSlT7hOEbWkYQqEOLvkwaulJjIAAcAAYMdM3ehpiPE942NZ0MRgQgFlfHh3yQAAmDFSBFNpIKpOdFVmXYeCDHhuRvOhSGPOAADFVDjLNZWvPMIBAGEYSAA) de Prettier funciona.

Reply:
Yes, there is a problem there. We need to find what rule is forcing Prettier to eliminate that space. We can take a look deep into a gardening day.